LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2012 Dave Grijalva
```

Comment:
I think we can just state (c) 2022 golang-jwt maintainers here

Reply:
Fixed